### PR TITLE
Add initial documentation

### PR DESCRIPTION
**Pull request by sleipnir**
I think it's important that we include some documentation to support dotnet C#. Many people may not even know that we provide support for this language as we have no reference to it in the documentation. So I am humbly forwarding an initial version of documentation.
Some work must be added later mainly on the gettingstarted page because, I do not explain details about the initial build. I think @nagytech can explain these details better later.

I was not able to explicitly add the reviewers because I am not part of the C# team but I think @nagytech and @pvlugter, maybe @marcellanz or @viktorklang can also take a look at this PR.

PS.: It is not part of this PR but maybe it would be interesting to include myself in the C# team since I periodically contribute here. It would be nice to have someone else anyway.

### CODE REVIEW

**Comment by sleipnir** [comment]
This PR also intends to close this related issue https://github.com/cloudstateio/dotnet-support/issues/4

File: docs/src/main/paradox/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# .Net
```
Comment (pvlugter) [inline docs/src/main/paradox/index.md]: Let's use `Cloudstate .Net` like the other projects.

**Review by pvlugter — COMMENTED** [review]
Good idea :+1:

I've added you to the @cloudstateio/dotnet-team 

And here's the deploy token encrypted for travis for this repo:

```
# Docs deploy token encrypted with: travis encrypt --pro -r cloudstateio/dotnet-support DEPLOY_DOCS_TOKEN={token}
secure: "v+u7GSntZkypS2JF9dohQFf43stc18wsWoWSPs265L2RHQHYrVUT5RjiDGmTZ4LSDT3QRJfXFqsuLIAJKo0mZDCGAzZg3Jb1wOznZk0UChWrRyuizQevqRMlhbTz+NBI+to+mAozrMX7n/MH+OGdYTOlmRdr382ar36fe733WbqIMHhPakFSzKD2itUMsiEzIr9E0KWMjdIzcvzXD0BdtnBVfWG8KuwLUl4UqWZp6nwMOWXXLJp6VK5mHbXC4zk32Yx4wsShLIkDtI46Nlp3USi7WPaqvjJWUdEnyTtBZXeG0lsZ8x7qPU9R/8YaePYlN1yuASuCN40vzbZk4aXXNfNvL1Cqs+U+ln4qsNkBs2L93yWN3DFLBfB+Sp/9QaGps1KW9AbZI9M7Na8sK3CsW0GAwMuvoDuDFHlh3QXoTQag0V+C3KcxRJwgd99QDejkFjQK7iyPRVBsb8rgHJOKYMlQRv+TzXiahRBwu+kqdu+ZZ4yY4FA7bzN3XM+Zb4PgvVpyYzNM0NfDuLGpXM6NoKiUXrdXPiE15iWxgNCpygKOwefYsexdCEfY2158NlpDoYaOtfRHDZSuPnuv0unM+jB2x1BHblD6xBESEto0m1+M1CFqBgIx9TV0+bgXIrsmuSch9VMkoPXFi2BeEBxViMBJrKxLEL77tYBVvKaTReU="
```

File: docs/src/main/paradox/index.md (hunk 1 above)
Comment (sleipnir) [reply docs/src/main/paradox/index.md]: Ok

**Pull request merged**